Scripts/UI/Base/UserInterface.gd

```diff
@@ -1,103 +1,76 @@
 extends Control
 class_name UserInterface
-## UserInterface Class: [br]
 ## A Base UI class to contain common UI features within the game such as:[br]
 ## - Controller Ui Support.[br]
 ## - viewport Text Reizing.[br]
 ## - Translation Switching (Unimplemented).
```

Comment:
Whenever we get to this, look at LocalizableString. Will help a lot.

Reply:
I thought about implementing that (hence the documentation) but, unfortunately I don't have any experience in that area so I wanted to play around with it in a separate project before attempting anything here.